system/lvol.py

```diff
@@ -119,7 +125,7 @@ def parse_lvs(data):
     for line in data.splitlines():
         parts = line.strip().split(';')
         lvs.append({
-            'name': parts[0],
+            'name': parts[0].replace('[','').replace(']',''),
```

Comment:
because cache pool lvols are by default hidden, you must show them with lvs -a and in the output they will show up like this [some_lv], try to create cached lv's see man lvmcache, this will at least enable possibility to create them with your module. i am preparing separate module for managing cached lv's because it is not so easy as normal lv , you cannot resize them while they are cached...